File: Bachelor/Semester4/Systems_for_Design_and_Implementation/C#/ProiectCShMPP/Networking/RpcProtocol/Response.cs
using System;

namespace Networking.RpcProtocol
{
    [Serializable]
    public class Response
    {
        public ResponseType Type
        {
            get;
            set;
        }

        public Object Data
        {
            get;
            set;
        }

        public override string ToString()
        {
            return Type+" "+Data;
        }
    }
}

File: Bachelor/Semester5/Parallel_and_Distributed_Programming/P1/Networking/src/main/java/rpcprotocol/Response.java
package rpcprotocol;

import java.io.Serializable;

public class Response implements Serializable {
    private ResponseType type;
    private Object data;

    public Response(ResponseType type) {
        this.type = type;
    }

    public Response(ResponseType type, Object data) {
        this.type = type;
        this.data = data;
    }

    public ResponseType getType() {
        return type;
    }

    public void setType(ResponseType type) {
        this.type = type;
    }

    public Object getData() {
        return data;
    }

    public void setData(Object data) {
        this.data = data;
    }

    @Override
    public String toString() {
        return "Response{" + "type=" + type +
                ", data=" + data +
                '}';
    }
}